# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/base/CacheConstants.h
# Commit: c2233b3830ef
# Full Hash: c2233b3830efd3e475a76443d22c07e73cda476a
# Author: Morgan Rae Reschenberg <mreschenberg@mozilla.com>
# Date: 2022-06-11 09:51:55
# Regressor Bug: 1758689
# File Overlap Count: 1
# Description:
#   Bug 1758689: Create and maintain a viewport cache for hittesting r=Jamie,emilio
#   
#   This hittesting approach is borrowed from our Android implementation.
#   We maintain a cache of ID's for accessibles which are visible in the viewport.
#   This cache is created with a call to `nsLayoutUtils::GetFramesForArea`, which
# ==============================================================================

diff -r 7a17841a34cc -r c2233b3830ef accessible/base/CacheConstants.h
--- a/accessible/base/CacheConstants.h	Fri Jun 10 23:49:24 2022 +0300
+++ b/accessible/base/CacheConstants.h	Fri Jun 10 21:10:18 2022 +0000
@@ -26,6 +26,7 @@
   static constexpr uint64_t ScrollPosition = ((uint64_t)0x1) << 11;
   static constexpr uint64_t Table = ((uint64_t)0x1) << 12;
   static constexpr uint64_t Spelling = ((uint64_t)0x1) << 13;
+  static constexpr uint64_t Viewport = ((uint64_t)0x1) << 14;
   static constexpr uint64_t All = ~((uint64_t)0x0);
 };
 